code/game/objects/items/reagent_containers/autoinjectors.dm

```diff
@@ -166,6 +178,10 @@
 	maptext_label = "Bi"
 
 /obj/item/reagent_container/hypospray/autoinjector/bicaridine/skillless
+	name = "bicaridine EZ autoinjector"
+	desc = "An EZ autoinjector loaded with 3 uses of Bicaridine and doesn't require any training, a common brute and circulatory damage medicine."
```

Comment:
```suggestion
	desc = "An EZ autoinjector loaded with 3 uses of Bicaridine, a common brute and circulatory damage medicine.  Doesn't require any training to use."
```
copy this change for every other autoinjector, otherwise the descriptions sound unnatural and out of place.

Reply:
Will do!